.github/workflows/docker-build.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
           fetch-depth: 0
       - name: set matrix
         id: set-matrix
-        run: echo "images=$(find docker -maxdepth 2 -mindepth 2 | jq --raw-input --slurp --compact-output 'split("\n")[:-1]')" >> $GITHUB_OUTPUT
+        run: echo "images=$(find docker -maxdepth 1 -mindepth 1 | jq --raw-input --slurp --compact-output 'split("\n")[:-1]')" >> $GITHUB_OUTPUT
```

Comment:
I think this line is overengineered at this point. Should this be rewritten somehow?

Reply:
I do something very similar [here](https://github.com/stjudecloud/cicd-workflows/blob/main/.github/workflows/app-nightly-scan.yaml#L19) with the difference being using `ls docker` would drop the `docker/` prefix. Haven't looked farther to see if that is used.

But the `jq` piece is good.

Reply:
Ok I think we _do_ use the `docker` prefix, so I propose we leave this for now.